style: Minor nits on the alignment properties.

emilio opened the pull request:
I'm going to touch this in a bit, let's do it a bit less painful.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/servo/servo/19850)
<!-- Reviewable:end -->

emilio committed:
style: Minor nits on the alignment properties.

I'm going to touch this in a bit, let's do it a bit less painful.

MozReview-Commit-ID: LhBNMkUXlUK

highfive commented:
Heads up! This PR modifies the following files:
 * @bholley: components/style/values/specified/align.rs
 * @canaltinova: components/style/values/specified/align.rs

highfive commented:
<img src="http://www.joshmatthews.net/warning.svg" alt="warning" height=20> **Warning** <img src="http://www.joshmatthews.net/warning.svg" alt="warning" height=20>

* These commits modify style code, but no tests are modified. Please consider adding a test!

emilio committed:
style: Rename AlignJustifyContent to ContentDistribution.

align-content and justify-content will have different types in a second.

MozReview-Commit-ID: 5JDeR5kXZNP

emilio committed:
style: Rename AlignJustifySelf to SelfAlignment.

This matches the spec term and, again, the two properties will have different
grammars soon.

MozReview-Commit-ID: 8f8JXj2NnCi

emilio commented:
r? @nox or @mbrubeck

nox commented on components/style/gecko/generated/structs.rs:
```diff
@@ -26590,8 +26590,8 @@ pub mod root {
-        #[link_name = "\u{1}_ZN10nsCSSProps34kAutoCompletionAlignJustifyContentE"]
-        pub static mut nsCSSProps_kAutoCompletionAlignJustifyContent:
+        #[link_name = "\u{1}_ZN10nsCSSProps34kAutoCompletionContentDistributionE"]
+        pub static mut nsCSSProps_kAutoCompletionContentDistribution:
```
Is this change intended? Please ELI5.

emilio replied:
Yeah, it is not, but it's harmless anyway.

nox reviewed (approved):
r=me with an explanation in the ticket, aren't the generated files you changed generated on the Gecko side?

emilio commented:
@bors-servo r=nox

 * I reverted the generated changes.

bors-servo commented:
:pushpin: Commit 711ea51 has been approved by `nox`

<!-- @bors-servo r=nox 711ea51d7b44f95362a366ccdb48a720791ea28c -->

bors-servo commented:
:hourglass: Testing commit 711ea51d7b44f95362a366ccdb48a720791ea28c with merge f28c77cb4969634f0559e70e06df1cac58846b2c...

bors-servo commented:
:broken_heart: Test failed - [arm32](http://build.servo.org/builders/arm32/builds/10745)

emilio commented:
@bors-servo retry

 * ARM build failed to checkout?

bors-servo commented:
:hourglass: Testing commit 711ea51d7b44f95362a366ccdb48a720791ea28c with merge ab00a3ec6668b4288ca1ba9bb18ab1b102d9b81b...

bors-servo commented:
:broken_heart: Test failed - [arm32](http://build.servo.org/builders/arm32/builds/10748)

emilio commented:
@bors-servo retry 

> command timed out: 1200 seconds without output running ['./mach', 'build', '--rel', '--target=arm-unknown-linux-gnueabihf'], attempting to kill

bors-servo commented:
:hourglass: Testing commit 711ea51d7b44f95362a366ccdb48a720791ea28c with merge e6470a1673670ef916a14e89a930335e02a9f63e...

bors-servo commented:
:broken_heart: Test failed - [arm32](http://build.servo.org/builders/arm32/builds/10750)

emilio commented:
@bors-servo retry

bors-servo commented:
:hourglass: Testing commit 711ea51d7b44f95362a366ccdb48a720791ea28c with merge 7354a32c19ca452748ee83452c03a4e15f11f242...

bors-servo commented:
:sunny: Test successful - [android](http://build.servo.org/builders/android/builds/10512), [arm32](http://build.servo.org/builders/arm32/builds/10754), [arm64](http://build.servo.org/builders/arm64/builds/10777), [linux-dev](http://build.servo.org/builders/linux-dev/builds/10806), [linux-rel-css](http://build.servo.org/builders/linux-rel-css/builds/7672), [linux-rel-wpt](http://build.servo.org/builders/linux-rel-wpt/builds/7689), [mac-dev-unit](http://build.servo.org/builders/mac-dev-unit/builds/10882), [mac-rel-css1](http://build.servo.org/builders/mac-rel-css1/builds/3280), [mac-rel-css2](http://build.servo.org/builders/mac-rel-css2/builds/3303), [mac-rel-wpt1](http://build.servo.org/builders/mac-rel-wpt1/builds/7546), [mac-rel-wpt2](http://build.servo.org/builders/mac-rel-wpt2/builds/7616), [mac-rel-wpt3](http://build.servo.org/builders/mac-rel-wpt3/builds/3376), [mac-rel-wpt4](http://build.servo.org/builders/mac-rel-wpt4/builds/3323), [windows-msvc-dev](http://build.servo.org/builders/windows-msvc-dev/builds/6188)
Approved by: nox
Pushing 7354a32c19ca452748ee83452c03a4e15f11f242 to master...
